### ISSUE TITLE
Support for aliases when importing command to CLI using `add_command`

### ISSUE THREAD

**Issue by tjzegmott** [click-contrib/click-aliases#8]
Do you plan on adding support for the following situation?

``` python
# cli.py
import click

from click_aliases import ClickAliasedGroup

from foo import foo


@click.group(cls=ClickAliasedGroup)
def cli():
    pass


cli.add_command(foo, aliases=['bar', 'baz', 'qux'])

if __name__ == "__main__":
    cli()

# foo.py
import click


@click.command()
def foo():
    """Run a command."""
    click.echo('foo')
```

**Comment by themagiulio** [comment]
@tjzegmott I added a PR #9. Hopefully the maintainer will merge it.

**Comment by rbonthond** [comment]
I've merged it

**Comment by themagiulio** [comment]
> When you get a chance, could you please release a new version on PyPI with this feature?

It would be nice to have this feature upstream. However, if you need this asap I suggest you to create the module with the new changes in your own project. As an example I leave here the module I've added to a cli I've created in the past months: https://github.com/themagiulio/memos-cli/blob/master/memos/click_aliases.py

Note: creating the module by yourself will prevent changes made upstream by the click-aliases creator. So I would consider this as a temporary fix.

**Comment by rbonthond** [comment]
I've published an 1.0.2 release. It's been years since I've done a release, so please check and let me know how it goes.
